newm-server-portal/src/main/kotlin/io/projectnewm/server/portal/PortalModule.kt

```diff
@@ -3,12 +3,16 @@ package io.projectnewm.server.portal
 import io.ktor.server.application.*
 import io.ktor.server.response.*
 import io.ktor.server.routing.*
+import io.projectnewm.server.portal.repo.PortalRepository
 
 @Suppress("unused")
 fun Application.portalModule() {
+    val repository = PortalRepository()
     routing {
-        get("/portal") {
-            call.respond(mapOf("Module" to "Portral"))
+        route("/portal/songs") {
```

Comment:
Is this endpoint intended to return songs belonging to the authenticated user?

Will we have a different one to see someone else's songs? Something like `/portal/artist/:someid/songs`?

Reply:
Yes, ultimately it will be something like that.